git_jira/git_jira.py

```diff
@@ -12,6 +12,13 @@
 
 MAX_RESULT = 5
 
+# Mapping of ticket types to branch prefixes
+TICKET_TYPE_TO_PREFIX = {
```

Comment:
I'd honestly move this to an environment variable, so everyone can define the mapping themselves.
`JIRA_TYPE_MAPPING="Story=feature,Task=feature"`
https://github.com/sloria/environs can read these maps.